README.md

```diff
@@ -30,10 +30,52 @@ like "scuddle" (i'm a kube cuddle person) - ergo: "scuttle".
 
 ### Installation
 
+**Homebrew**:
+
+> Currently only x86 linux/mac are supported.
+
+Install sctl
+```
+brew tap vapor-ware/formula
+brew install vapor-ware/formula/sctl
+```
+
+
+You'll also need the google cloud sdk to do stuff with kms using scuttle
+```
+brew install google-cloud-sdk
+```
+
+**Snap Packages**:
+
+> Currently only x86 linux are published.
+> Note: while we continue to refine the security policy of sctl snap packaging
+> there may be cases where it does not work as expected:
+>
+> - in $HOME, sctl does not have access to dotfiles.
+> - in $HOME/*/ sctl does not have access to dotfiles.
+> - In $HOME/*/* sctl has access and will be able to read/write to its state
```

Comment:
I believe this will render the `*/*` portion of this as an italics `/`. perhaps wrap these paths in backticks?